tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/TagMap.java

```diff
@@ -108,7 +109,7 @@ private static String[] toArray(Map<String, String> data) {
             int valuesIndex = 2 * i;
             String key = keys[i];
             values[valuesIndex] = key;
-            values[valuesIndex + 1] = data.get(key);
+            values[valuesIndex + 1] = Strings.nullToEmpty(data.get(key));
```

Comment:
this is a slight behavior change where previously we may have allowed null values through, we now convert to an empty string tag value `""`, though previously we would likely trigger a `NullPointerException` when copying the tag map to an `ImmutableMap<String, String>`

Reply:
I don't like silently replacing `null` with the empty string. These are semantically different values and this implicit conversion can be confusing for consumers.